delocate/cmd/common.py

```diff
@@ -70,8 +70,15 @@
 delocate_parser.add_argument(
     "--sanitize-rpaths",
     action="store_true",
-    help="Remove absolute and relative rpaths from binaries",
+    help="Remove absolute and relative rpaths from binaries (default)",
 )
+delocate_parser.add_argument(
+    "--no-sanitize-rpaths",
+    action="store_false",
+    dest="sanitize_rpaths",
+    help="Don't remove absolute and relative rpaths from binaries",
+)
+delocate_parser.set_defaults(sanitize_rpaths=True)
```

Comment:
Move `delocate_parser.set_defaults(sanitize_rpaths=True)` to `default=True` in the `--sanitize-rpaths` argument.